lib/presentation/crowd_action/pages/widgets/in_spotlight_header.dart

```diff
@@ -42,94 +42,95 @@ class _InSpotLightHeaderState extends State<InSpotLightHeader> {
       child: BlocBuilder<SpotLightBloc, SpotLightState>(
         builder: (context, state) {
           return state.maybeWhen(
-              fetchingCrowdSpotLightActions: () {
-                return const CircularProgressIndicator(
-                  color: kAccentColor,
-                );
-              },
-              spotLightCrowdActionsError: (_) {
-                // TODO - Implement screen with missing spot lights
-                return const Text("Error");
-              },
-              spotLightCrowdActions: (_pages) {
-                return Column(
-                  mainAxisSize: MainAxisSize.min,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    const SizedBox(
-                      height: 35,
+            fetchingCrowdSpotLightActions: () {
+              return const CircularProgressIndicator(
+                color: kAccentColor,
+              );
+            },
+            spotLightCrowdActionsError: (_) {
+              // TODO - Implement screen with missing spot lights
```

Comment:
We need to tell this to Niklas so he knows we're missing this.

Reply:
This issue was raised by Isaac in yesterday's Design meet. Niklas has added this to his Todo.